### PR TITLE
fixes #4 - Added function to access search result aggregation

### LINKED ISSUE #4: Add function to access search result aggregation
Presently there is no function to access search result aggregation.

### PR DESCRIPTION
**Pull request by djmgit**
fixed issue #4.Added function to access search result aggregation
API service.

### CODE REVIEW

File: lib/loklak.js [hunk 1]
```diff
@@ -127,4 +127,37 @@ Loklak.prototype.suggest = function (params, cb) {
...
+	if (params.count !== undefined) {
+        apiUrl += '&count=' + params.count;
+	}
+	if (params.fields !== undefined) {
+		apiUrl += '&fields=' + params.fields.join(',');
+	}
+	if (params.limit !== undefined) {
+		apiUrl += '&limit=' + params.limit;
+	}
+	console.log(apiUrl);
```
Comment (sudheesh001) [inline lib/loklak.js]: You can remove this console statement

File: lib/loklak.js [hunk 2]
```diff
@@ -127,4 +127,37 @@ Loklak.prototype.suggest = function (params, cb) {
...
+	this.httpClient.get(apiUrl, function (err, res, body) {
+		if (err) {
+			cb({});
+		} else {
+			cb(body);
+		}
+	});
+}
+
 module.exports = new Loklak();
```
Comment (sudheesh001) [inline lib/loklak.js]: Please add a new line after the EOF

**Review by sudheesh001 — CHANGES_REQUESTED** [review]
Looks good but a few minor changes.

File: lib/loklak.js (hunk 1 above)
Comment (djmgit) [reply lib/loklak.js]: yeah sure!! I missed it !

**Comment by djmgit** [comment]
Removed console statement and added EOF line

**Review by sudheesh001 — APPROVED** [review]
Looks good!

**Pull request merged**